_pages/en_US/bannerbomb3.txt

```diff
@@ -22,43 +22,41 @@ If you have a Taiwanese console (indicated by a T after the system version), fol
+  + `<ID0>` is the 32-letter folder name that you copied in [Seedminer](seedminer)
+  + `<ID1>` is a 32-letter folder inside of the `<ID0>`
   + This will download an exploit DSiWare called `F00D43D5.bin` inside of a zip file (`BannerBomb3.zip`)
 1. If your console is powered on, power off your console
 1. Insert your SD card into your computer
 1. Navigate to `Nintendo 3DS` -> `<ID0>` -> `<ID1>` on your SD card
   + `<ID0>` is the 32-letter folder name that you copied in [Seedminer](seedminer)
   + `<ID1>` is a 32-letter folder inside of the `<ID0>`
-  ![]({{ "/images/screenshots/bb3/dsiware-location-1.png" | absolute_url }}){: .notice--info}
 1. Create a folder named `Nintendo DSiWare` inside of the `<ID1>`
   + If you already had the folder *and* there are any existing DSiWare backup files (`<8-character-id>.bin`) inside, copy them to your PC and remove them from your SD card
-1. Copy the `F00D43D5.bin` file from `BannerBomb3.zip` to the `Nintendo DSiWare` folder
+1. Copy the `F00D43D5.bin` file from the `DSIWARE_EXPLOIT.zip` to the `Nintendo DSiWare` folder
+1. Copy `bb3.bin` from the `DSIWARE_EXPLOIT.zip` to the root of your SD card
+1. Reinsert the SD card into your device
 
-![]({{ "/images/screenshots/bb3/dsiware-location-2.png" | absolute_url }})
-{: .notice--info} 
 
-#### Section II - BannerBomb3
+#### Section II - Button Check
 
-In this section, you will trigger the BannerBomb3 exploit using the DSiWare Management menu and copy the resulting file dump to your computer so that you can use it on the next page.
+1. With your device still powered off, hold the following buttons: (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A), and while holding these buttons together, power on your device
+  + Keep holding the buttons until the device boots into Safe Mode (a "system update" menu)
 
-1. Reinsert your SD card into your device
-1. Power on your device
-1. Launch System Settings on your device
-1. Navigate to `Data Management` -> `DSiWare` -> `SD Card` ([image](/images/screenshots/bb3/dsiware-management.png))
-  + Your system should flash Magenta (pink/purple) and then crash a few seconds later. This means it worked
-1. Power off your device
-1. Insert your SD card into your computer
-1. You should now have `42383841.bin` on the root of your SD card. Copy this file to somewhere on your computer, as you will use it on the next page
-1. Navigate to `Nintendo 3DS` -> `<ID0>` -> `<ID1>` -> `Nintendo DSiWare` on your SD card
-1. Delete `F00D43D5.bin` from your Nintendo DSiWare folder and from your computer. This file will not be needed anymore
+### Next steps
+
+If your device booted into Safe Mode, continue to [Installing boot9strap (USM)](installing-boot9strap-(usm))
 
+If you're unable to get into Safe Mode after multiple attempts, one of your buttons may be failing or broken. If this is the case, you will need to continue to [Installing boot9strap (Fredtool)](installing-boot9strap-(fredtool))
```

Comment:
these should be notices

Reply:
I rewrote this a bit to somewhat match what Seedminer is doing.